app/controllers/revise_auth/password_controller.rb

```diff
@@ -1,24 +1,18 @@
 class ReviseAuth::PasswordController < ReviseAuthController
-  before_action :validate_current_password, only: [:update]
+  before_action :authenticate_user!
 
   def update
     if current_user.update(password_params)
-      flash[:notice] = I18n.t("revise_auth.password_changed")
+      redirect_to profile_path, notice: I18n.t("revise_auth.password_changed")
+    else
+      flash[:alert] = I18n.t("revise_auth.incorrect_password")
+      render "revise_auth/registrations/edit", status: :unprocessable_entity
```

Comment:
Nice work!
Suggestions: are we not following the single quotes convention for string?

Reply:
We are following standardrb.